App/Logic/ExposureDetectionLogic.swift

```diff
@@ -59,11 +59,57 @@ extension Logic.ExposureDetection {
         isUserCovidPositive: state.user.covidStatus.isCovidPositive,
         forceRun: self.forceRun
       ))
-
+        
+        //tupla = (countryId,latestProcessedKeyChunkIndex, lastDetectionDate)
+      var exposureDetectionCountries: [(String, Int?, Date?)] = []
+      let outcomeEU: ExposureDetectionOutcomeEU
+        
+      if !state.exposureDetectionEU.isEmpty {
+        
+        for exposureDetectionCountry in state.exposureDetectionEU {
+            exposureDetectionCountries.append(
+                (exposureDetectionCountry.countrySelection.country.countryId, exposureDetectionCountry.exposureDetectionState.latestProcessedKeyChunkIndex, exposureDetectionCountry.exposureDetectionState.lastDetectionDate))
+            }
+        outcomeEU = try await(
+          context.dependencies.exposureDetectionExecutor.executeEU(
+          exposureDetectionPeriod: self.type.detectionPeriod(using: state.configuration),
+          lastExposureDetectionDate: state.exposureDetection.lastDetectionDate,
+          exposureDetectionConfiguration: state.configuration.exposureConfiguration,
+          exposureInfoRiskScoreThreshold: state.configuration.exposureInfoMinimumRiskScore,
+          userExplanationMessage: L10n.Notifications.AppleExposureNotification.message,
+          enManager: context.dependencies.exposureNotificationManager,
+          tekProvider: context.dependencies.temporaryExposureKeyProvider,
+          now: context.dependencies.now,
+          isUserCovidPositive: state.user.covidStatus.isCovidPositive,
+          forceRun: self.forceRun,
+          exposureDetectionEU: exposureDetectionCountries
+          ))
+        
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/ExposureDetectionLogic.swift;
  Line: 87;
-->

- :warning: swiftlint (0.39.2): Lines should not have trailing whitespace. (trailing_whitespace)